### DISCUSSION TITLE
New G6 Sensor repeatedly marked as stopped

### DISCUSSION THREAD

**Discussion by kaindl** [NightscoutFoundation/xDrip#2085] [Q&A]
Hi,
I had to replace my G6 sensor today in the morning (not replacing the transmitter, its the 4th or 5th sensor with that transmitter) and I am having problems since then: Sensor started normal with 2 hours warming up, got 1 reading after the warmup, then xdrip told me, Sensor start failed.
I could see in the log, that it still got readings every 5 minutes, but it ignored them, because no sensor is active. After starting the same sensor again in xdrip, I get 2-6 readings and it tells me again, the sensor is stopped.

Also reactivated the original Dexcom Reader after a few tries.. It gets readings every 5 minutes without any problems..

I attached screenshots of the system status page and of the log viewer showing one of the unsuccessfull sensor starts.
![Screenshot_20220423-233822](https://user-images.githubusercontent.com/8314747/164946899-63546705-9abd-41d3-85f0-df2b7450733d.png)
![Screenshot_20220423-233854](https://user-images.githubusercontent.com/8314747/164946907-428705c2-c7a1-4c99-afc7-54ee9553ac92.png)
![Screenshot_20220423-234211](https://user-images.githubusercontent.com/8314747/164946913-c6a41a13-c7c0-4664-b982-dcc00944f8ec.png)

**Comment by Navid200** [comment]
Please note that if for any reason a sensor session really stops on that transmitter, you cannot just tap on start sensor and expect it to work.
I can see in the logs that at 13:04 sensor stopped and at 13:08, you started the sensor again.  That will never work.  Just because xDrip starts a warm-up counter does not mean that things are working.

But, that doesn't answer why the sensor stopped in the first place.
Can you post a screenshot of the main screen and tell us which readings you got  from this sensor?

**Reply by kaindl** [reply]
As I wrote, the session was not really stopped on the transmitter:
* I still could see incoming readings in the log, but they were ignored, as xdrip had no active sensor session
* The sensor was still active and showing readings every 5 minutes on the Dexcom Receiver

So the question is, why did xdrip mark the session as stopped, when it was still active on the transmitter?

But it literally solved itself over night:
When I took the screenshots, the main screen told me to start a new sensor or check settings, as it did the whole day. When I went to bed a little bit later, it just told ne, there were no readings, but nothing about the no sensor anymore.
I restarted the collector and went to bed and in the morning it was running without any problems :)

So is there still anything we can do to find out, what happened to my xdrip yesterday?

**Reply by Navid200** [reply]
I don't think so.

There are some messages that are valid for G5 but not for G6.  We are cleaning those out.  The last xDrip release is an example.  I myself am testing it now as I have a sensor start coming up today and will see if the messages have improved or not.

You are in a good shape considering you have a receiver.

**Reply by tolot27** [reply]
> So is there still anything we can do to find out, what happened to my xdrip yesterday?

Yes, please downgrade to [2022.04.20](https://github.com/NightscoutFoundation/xDrip/releases/tag/2022.04.20) and report back if the problem still exists.

**Reply by kaindl** [reply]
As the problem vanished last night, that could be difficult ;)
But if it happens again with the next sensor, I will try that.

**Reply by jamorham** [reply]
I think the main issue here is that we get a session start failure message but apparently the session doesn't fail to start.

**Reply by enginemr2** [reply]
I had a similar problem. One night for some reason my Dexcom receiver went wild with a low BG level. Of course, I had been sleeping. I woke and looked at my watch, my standalone Dexcom data collector. It showed that all data had stopped some time ago. I have used this watch for the last 5 years and it has been very reliable, but I still figured there was a problem. I immediately checked my BG level with my meter. It showed that I was at 115 mg/dl so definitely not low. At that time, I again looked at the Dexcom and it showed a BG level of less than 45. A few minutes later the Dexcom receiver then showed that the sensor had failed. 
Checking the Dexcom data it showed that the sensor jumped from a normal value, 115, and then down to 45 and back to a much higher value. I believe that the xDrip smarts detected there was a problem so stopped displaying the data on the watch. When I checked the data on my smartphone, that is paired with my watch, it did show that the data was not valid. 
In this case it is always required to verify any questionable data from a sensor. After that I immediately changed the sensor, and everything is back to normal. 
My watch is the Sony Smartwatch 3 which I have been using since the Dexcom G5 and now the G6, a very reliable data collector every 5 minutes. Yes, I have three backup watches.
